tools/torch/test.lua

```diff
@@ -343,40 +343,59 @@ else
     end
     counter = 1 -- here counter is set, so that predictBatch() method displays only the predictions of first image
     predictBatch(inputs, model)
+
     if opt.visualization=='yes' then
-        local filename = paths.concat(opt.save, 'vis.h5')
```

Comment:
@TimZaman Shoot, yeah I've been using Atom but I haven't installed any code linting packages :p . I will fix this in my next commit.